Maple2.Server.Game/PacketHandlers/UserSyncHandler.cs

```diff
@@ -39,7 +39,9 @@ public override void Handle(GameSession session, IByteReader packet) {
             stateSyncs[i] = stateSync;
 
             packet.ReadInt(); // ClientTicks
-            packet.ReadInt(); // ServerTicks
+            if (playerState != ActorState.WeddingEmotion) {
```

Comment:
tf?

Reply:
This is not totally correct but the packet is smaller if the state is WeddingEmotion. I'm not sure what it is